### ISSUE TITLE
No Fallldamage when you land on the edge of a block.

### ISSUE THREAD

**Issue by TBlueF** [SpongePowered/SpongeForge#802]
**Tested successfully with:**
_Minecraft: 1.8.9
SpongeAPI: 4.2.0-SNAPSHOT-1f5dac2
SpongeForge: 1.8.9-1890-4.2.0-BETA-1625
Minecraft Forge: 11.15.1.1890_
**and:**
_Minecraft: 1.10.2
SpongeAPI: 5.0.0-SNAPSHOT-a13a8d6
SpongeForge: 1.10.2-2026-5.0.0-BETA-1632
Minecraft Forge: 12.18.1.2026_

**How to reproduce:**
Build an L-Shape like this: http://imgur.com/KvC6A2R
Make it as high as you want and place yourself on top. Then go into survival-mode and sneak to one corner of the pillar so you can jump down on the very edge of the L-Shape-Base.
Basically you jump from somewhere very high down and land on an edge like this: http://imgur.com/OmL7pcm

**What i would expect to happen:**
The Player takes massive fall-damage and dies.

**What actually happens:**
The Player takes no fall-damage at all.

**Additional notes:**
This only happens if the block that is directly below the player when he lands is **AIR**
If it's any other block (even one without a collision-box, like tall-grass) the player takes the damage.
This does not happen without Sponge installed.

**Comment by ST-DDT** [comment]
Does this occur in SpongeVanilla as well?

**Comment by TBlueF** [comment]
Yes it does!
I just tested it with a new Server:
_Minecraft: 1.8.9
SpongeAPI: 4.2.0-SNAPSHOT-79f9c8e
SpongeVanilla: 1.8.9-4.2.0-BETA-348_

Should i close it here then and create a new issue it in the Common Repo?

**Comment by Aaron1011** [comment]
@TBlueF: I'm able to reproduce this - thanks for documenting it in so much detail! I've had an intermittent issue where I haven't been taking fall damage, but was never able to determine precisely what was causing it.

**Comment by Aaron1011** [comment]
Closed in https://github.com/SpongePowered/SpongeCommon/commit/8ed0112572af76df6140636c4babdf1f8b84f1b8.
